internal/highway/proto/v1/highway.proto

```diff
@@ -82,4 +82,18 @@ service Highway {
   //
   // Deletes the given blob from the calling node's service IPFS storage.
   rpc RemoveBlob(sonrio.highway.v1.MsgRemoveBlob) returns (sonrio.highway.v1.MsgRemoveBlobResponse) {}
+
+  // QueryParams
+  //
+  //
+  rpc QueryParams(sonrio.sonr.registry.QueryParamsRequest) returns (sonrio.sonr.registry.QueryParamsResponse) {}
+
+  // QueryWhoIs
+  //
+  //
+  rpc QueryWhoIs(sonrio.sonr.registry.QueryWhoIsRequest) returns (sonrio.sonr.registry.QueryWhoIsResponse) {}
+
+  // QueryWhoIsAll
+  //
+  rpc QueryWhoIsAll(sonrio.sonr.registry.QueryWhoIsAllRequest) returns (sonrio.sonr.registry.QueryAllWhoIsResponse) {}
```

Comment:
Please remove new proto definitions as this is not something we wish to expose over grpc

Reply:
done